Build/start.sh

```diff
@@ -4,6 +4,7 @@ set -e
 SOMHOSTNAME="MistySOM-V2L"
 LOCALCONF="${WORK}/build/conf/local.conf"
 YOCTODIR="/home/yocto/rzv_vlp_v3.0.0/" 
+PASSWD="yocto"
```

Comment:
This is not being used anymore. Let's delete it.

Reply:
> So this line can be deleted?

> This is not being used anymore. Let's delete it.

Both lines removed